pkg/logger/logger.go

```diff
@@ -13,10 +13,12 @@ func Get() *zap.Logger {
 	config.DisableStacktrace = true
 	config.InitialFields = map[string]any{"service": "sprayproxy"}
 	config.EncoderConfig.EncodeTime = utcRFC3339TimeEncoder
+	config.InitialFields = map[string]interface{}{"audit": "true"}
```

Comment:
`config.InitialFields` is set on line 14 and here you override it. Instead you should add the `audit` key there.
```suggestion
	config.InitialFields = map[string]any{"service": "sprayproxy", "audit": "true"}
```

Reply:
silly me.